### PR TITLE
Do not override sensorToWorldMatrix if animGraph is loading or failed to load

### PR DESCRIPTION
**Pull request by druiz17**
Before the camera is updated.. Interface will call ```MyAvatar::postUpdate``` if the ```skeletonModel``` does not have an ```animNode``` it will call ```initAnimGraph``` which will reset ```bodySensorMatrix``` and ```sesnsorToWorldMatrix``` after the physics results have been harvested . The result of this will be undesired camera behavior that will make the user sick.
This PR makes sure that ```MyAvavatar::postUpdate``` does not override the ```sensorToWorldMatrix```  
and also set the load priority of the animGraphLoader

ticket - https://highfidelity.fogbugz.com/f/cases/11490/When-the-avatar-is-in-a-T-pose-the-view-in-the-HMD-no-longer-tracks-the-head

### CODE REVIEW

**Comment by druiz17** [comment]
Test plan
- go to a heavy loaded domain
- put on your HMD
- make sure that when you turn your head while the domain is loading..the world does not pan in the opposite direction

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6551/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6551/HighFidelity-Beta-PR12239-071f7d54d9740f58a84da78e3add16789af5d590.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6551/HighFidelity-Beta-PR12239-071f7d54d9740f58a84da78e3add16789af5d590.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6567/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6567/HighFidelity-Beta-PR12239-e467286a1c8c9607177bdf08b3acb01dba1d3c6f.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6570/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6570/HighFidelity-Beta-PR12239-b7ba7862aa929a501a7fec542974d6ce3ebb4d5c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6570/HighFidelity-Beta-PR12239-b7ba7862aa929a501a7fec542974d6ce3ebb4d5c.exe

**Comment by conklin94122** [comment]
@druiz17 @hyperlogic Did Dante make the requested changes? Should the change request be lifted? I don't want to have the PR tested if the changes still need to go in and we'll need to retest.

**Comment by druiz17** [comment]
@conklin94122 I made the changes, Tony just needs to review them.

**Comment by conklin94122** [comment]
@hyperlogic Please review when you can.

**Comment by conklin94122** [comment]
Removed QA Ready. Please put it back when ready for testing.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6884/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6884/HighFidelity-Beta-PR12239-d593e7107c44bc319005a1d384f8f2fd577dfcb2.exe

**Comment by druiz17** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6888/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6888/HighFidelity-Beta-PR12239-d593e7107c44bc319005a1d384f8f2fd577dfcb2.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6888/HighFidelity-Beta-PR12239-d593e7107c44bc319005a1d384f8f2fd577dfcb2.dmg

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -1876,26 +1874,33 @@ void MyAvatar::initAnimGraph() {
...
+    _cauterizationNeedsUpdate = true;
+    disconnect(&(_skeletonModel->getRig()), SIGNAL(onLoadComplete()), this, SLOT(animGraphLoaded()));
+}
+
 void MyAvatar::postUpdate(float deltaTime, const render::ScenePointer& scene) {
 
     Avatar::postUpdate(deltaTime, scene);
 
-    if (_skeletonModel->isLoaded() && !_skeletonModel->getRig().getAnimNode()) {
+    if (_skeletonModel->isLoaded() && !_skeletonModel->getRig().getAnimNode() && _initHeadBones) {
```
Comment (hyperlogic) [inline interface/src/avatar/MyAvatar.cpp]: It's more common for init booleans to be true when something is initialized and false when they are not.  can you rename this to _shouldInitHeadBones? This makes it more readable.  If _shouldInitHeadBones is true then we should call initHeadBones().  But I'm not holding up the PR for this.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6929/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6929/HighFidelity-Beta-PR12239-e87067f93d0841312c540073b755100ceb458f48.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6929/HighFidelity-Beta-PR12239-e87067f93d0841312c540073b755100ceb458f48.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7134/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7134/HighFidelity-Beta-PR12239-1a2533257574531b23c1af9994ca159d3ded2ffd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7134/HighFidelity-Beta-PR12239-1a2533257574531b23c1af9994ca159d3ded2ffd.exe

File: interface/src/avatar/MyAvatar.h [hunk 2]
```diff
@@ -817,6 +819,7 @@ private slots:
     bool _enableDebugDrawIKConstraints { false };
     bool _enableDebugDrawIKChains { false };
     bool _enableDebugDrawDetailedCollision { false };
+    bool _shouldInitHeadBones { true };
```
Comment (hyperlogic) [inline interface/src/avatar/MyAvatar.h]: this can be removed now.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7158/app_12239_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7158/HighFidelity-Beta-PR12239-45f09dde96c55e8982840bdb7bea57f61dcdb21f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7158/HighFidelity-Beta-PR12239-45f09dde96c55e8982840bdb7bea57f61dcdb21f.exe

**Comment by sbennett77** [comment]
Currently Testing.

**Comment by sbennett77** [comment]
Passed.

@conklin94122

**Pull request merged**